src/composables/Project.js

```diff
@@ -122,10 +121,9 @@ export async function fetchGit(project) {
         password: project.git.token,
       }),
       corsProxy: process.env.CORS_ISOMORPHIC_BASE_URL,
-    });
+    }).catch(() => Promise.resolve());
```

Comment:
no error management, why ?

Reply:
we got several types of issue with the fetch previously (depend on the git function used). Could you check that again during your tests ? if there is no issue anymore, we will delete this. Otherwise, could you comment here the error name and condition here, so we can make a more precise error management ?